provider/doc_edits.go

```diff
@@ -88,3 +89,26 @@ var substituteRandomSuffix = (func() tfbridge.DocsEdit {
 		},
 	}
 })()
+
+// Docs discovery gets tripped up on `member/members` fields for IAM-type properties and doesn't align the content correctly.
+
+var memberRegexp = regexp.MustCompile("`member/members`")
+
+var rewritemembersField = tfbridge.DocsEdit{
+	Path: "*iam.html.markdown",
+	Edit: func(path string, content []byte) ([]byte, error) {
+		membersByte := []byte("`members`")
+		memberByte := []byte("`member`")
+		var returnContent []byte
+		membersContent := memberRegexp.ReplaceAllLiteral(content, membersByte)
+		memberContent := memberRegexp.ReplaceAllLiteral(content, memberByte)
+		// Because the IamBinding property matches to a `members` field, while the `IAMMember` property matches to a
```

Comment:
As mentioned [on the issue](https://github.com/pulumi/pulumi-gcp/issues/1920#issuecomment-2057412960), the bridge cannot find a `member` _or_ a `members` field, because it is presented in the incoming docs as `member/members`. There is no mechanic in the bridge that will separate out two field names presented in this way, nor should there be because it's not a common pattern as far as I'm aware.
A find-and-replace for the regex only solves for one of the two possible cases, and a find-and-insert is significantly more difficult to do, so I'm duplicating the docs content.